terraform.tfvars.example

```diff
@@ -102,4 +102,6 @@ attached_drg_id = null
 #subnets = {
 #  sub1 = {name = "subnet1",cidr_block = "10.0.4.0/24"}
 #  sub2 = {cidr_block="10.0.5.0/24",type="private"}
+#  sub3 = {cidr_block="10.0.6.0/24",availability_domain="Uocm:PHX-AD-1"}
```

Comment:
see my comment about looking these values up in your other PR

Reply:
This has been fixed via other PR.

Reply:
What is the other PR?